lang/简体中文.json

```diff
@@ -10,7 +10,7 @@
         "no_campaign": "没有可用于掉宝的活动，等待一个有效的掉宝活动..."
     },
     "login": {
-        "unexpected_content": "返回的内容类型有误通常是由于被重定向.您需要登陆才能联网吗？",
+        "unexpected_content": "返回的内容类型有误通常是由于被重定向.您需要设置VPN或者确认登陆页面才能联网吗？",
```

Comment:
I have include both VPN setting and login confirmation i the notification

Reply:
Looks cool!

@Suz1e Does this translation work for you?

Reply:
yes